ledger/apply/application_test.go

```diff
@@ -814,6 +814,15 @@ func TestAppCallClearState(t *testing.T) {
 	a.Equal(0, len(br.AppLocalStates))
 	a.Equal(basics.StateSchema{}, br.TotalAppSchema)
 	a.Equal(basics.EvalDelta{GlobalDelta: gd}, ad.EvalDelta)
+
+	b.ResetWrites()
+	b.pass = true
+	b.err = nil
+	logs := map[basics.AppIndex][]string{appIdx: {"a"}}
```

Comment:
What's the program actually being run here?  It should do at least two logs, so we can test that they both appear.

Also test that a program with 33 logs fails.  And one with two 513 byte logs.

Reply:
I think I retract this.  As you say, these checks happen at a higher layer.